Bottom: 9dc8c8e4fe469792f6a0923b110a89551cfe4e47
Top:    4b5fd3415a42d90de80e6225aa50dc8a2e59c6eb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-11 15:31:32 +0000

Refresh of reduce-the-redundant-work-done

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 0b1dc37..3b8c201 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1101,14 +1101,17 @@ label_contour (PLINE * a)
           continue;
         }
 
+#if 0
       if (NODE_LABEL (cur) != UNKNWN)
         {
+          fprintf (stderr, "Not expected...\n");
           /* This might get hit if the last cross vertex labelling
            * pass was a bit keen, and labeled past its own edges?
            */
           label = NODE_LABEL (cur);
           continue;
         }
+#endif
 
       if (label == INSIDE || label == OUTSIDE)
         {
